### PR TITLE
Add support for the AUAV X2.1 Flight Controller

### PR DESCRIPTION
**Pull request by pkocmoud**
Don - Please review my boardFallback entries.

Note - Line location was chosen to fall in productID numeric order.

### CODE REVIEW

File: src/VehicleSetup/FirmwareUpgradeController.cc [hunk 1]
```diff
@@ -407,6 +418,8 @@ QHash<FirmwareUpgradeController::FirmwareIdentifier, QString>* FirmwareUpgradeCo
     case Bootloader::boardIDAeroCore:
         return &_rgAeroCoreFirmware;
     case Bootloader::boardIDMINDPXFMUV2:
```
Comment (davids5) [inline src/VehicleSetup/FirmwareUpgradeController.cc]: Duplicate case it should be ```boardIDAUAVX2_1```

**Comment by DonLakeFlyer** [comment]
All looks good. Could you try connecting and flashing on Windows with a build which has the fallbacks for you boards removed. I think they may not be needed. If both work fine without them you can remove them and this should be good to go.

**Comment by pkocmoud** [comment]
Removed the extra entries, tested good. Can you please merge?

**Comment by pkocmoud** [comment]
@DonLakeFlyer Please merge this.

**Comment by dagar** [comment]
I don't have this hardware, but the changes look correct.

**Pull request merged**